app/actors/hyrax/actors/create_with_files_actor.rb

```diff
@@ -39,7 +39,7 @@ def validate_files(files, env)
         # @return [TrueClass]
         def attach_files(files, env)
           return true unless files
-          AttachFilesToWorkJob.perform_later(env.curation_concern, files)
+          AttachFilesToWorkJob.perform_later(env.curation_concern, files, env.attributes.to_h.symbolize_keys)
```

Comment:
The new arg here does what line 43 of the `AttachFilesActor` did. It makes sure that e.g. embargo attrs are available for FileSets in the actor stack.